test/integration/sequelize.test.js

```diff
@@ -527,6 +527,16 @@ describe(Support.getTestDialectTeaser('Sequelize'), () => {
       });
     });
 
+    if (dialect === 'mssql') {
+      it('array in `bind` property working', function () {
+        let logSql;
+        return this.sequelize.query({query: 'select $prop', bind: {prop: [1]}}, {type: this.sequelize.QueryTypes.SELECT, logging(s) {logSql = s;}}).then(result => {
+          expect(result).to.deep.equal([{'': 1}]);
+          expect(logSql.indexOf('@prop_0')).to.be.above(-1);
```

Comment:
Asserting on log statements is a bad practice, if you want to assert the SQL use an SQL unit test